lms/templates/navigation.html

```diff
@@ -36,8 +36,8 @@
 % endif
 </%block>
 
-  <header class="global ${"slim" if course else ""}" aria-label="${_('Global Navigation')}">
-    <nav>
+  <header id="global-navigation" class="global ${"slim" if course else ""}" aria-label="${_('Global Navigation')}">
+    <nav aria-labelledby="global-navigation">
```

Comment:
Let's remove the aria-label from the ```<header>``` element and put it on the ```<nav>``` element.  No need to include the word "Navigation" as that will automatically be appended by the screen reader.